test/unit/deadline_job_attachments/test_upload.py

```diff
@@ -2122,6 +2122,24 @@ def test_get_asset_groups(
                 {},  # File System Location (SHARED type)
                 [],
             ),
+            (
+                {"c:\\USERNAME\\DOCS\\inputs\\input1.txt"},  # input paths
+                {"C:\\username\\docs\\outputs"},  # output paths
```

Comment:
question: Do we need/want test cases where the output path uses a lowercase `c`?

Reply:
Sure, I can add it